Sources/BraintreeVenmo/Models/VenmoCreatePaymentContextGraphQLBody.swift

```diff
@@ -0,0 +1,149 @@
+import Foundation
+
+// swiftlint:disable nesting
+/// The POST body for graph QL `mutation CreateVenmoPaymentContext`
+struct VenmoCreatePaymentContextGraphQLBody: Encodable {
+    
+    var query: String
+    var variables: Variables
+    
+    init(request: BTVenmoRequest, merchantProfileID: String?) {
+        // swiftlint:disable:next line_length
+        self.query = "mutation CreateVenmoPaymentContext($input: CreateVenmoPaymentContextInput!) { createVenmoPaymentContext(input: $input) { venmoPaymentContext { id } } }"
+        self.variables = Variables(request: request, merchantProfileID: merchantProfileID)
+    }
+    
+    
+    struct Variables: Encodable {
+        
+        let input: InputParameters
+        
+        init(request: BTVenmoRequest, merchantProfileID: String?) {
+            self.input = InputParameters(
+                paymentMethodUsage: request.paymentMethodUsage.stringValue,
+                merchantProfileID: merchantProfileID,
+                customerClient: "MOBILE_APP",
+                intent: "CONTINUE",
+                isFinalAmount: request.isFinalAmount.description,
+                displayName: request.displayName,
+                paysheetDetails: Variables.InputParameters.PaysheetDetails(
+                    collectCustomerBillingAddress: request.collectCustomerBillingAddress,
+                    collectCustomerShippingAddress: request.collectCustomerShippingAddress,
+                    transactionDetails: Variables.InputParameters.PaysheetDetails.TransactionDetails(
+                        lineItems: request.lineItems?.map { item in
+                            Variables.InputParameters.PaysheetDetails.LineItem(
+                                quantity: item.quantity,
+                                unitAmount: item.unitAmount,
+                                name: item.name,
+                                kind: item.kind.rawValue,
+                                unitTaxAmount: item.unitTaxAmount ?? "0",
```

Comment:
This looked to be the default behavior in `BTVenmoClient` so persisted this logic:
```swift
if item.unitTaxAmount == nil || item.unitTaxAmount?.isEmpty == true {
    item.unitTaxAmount = "0"
}
```